apps/web-giddh/src/app/shared/header/header.component.scss

```diff
@@ -62,7 +62,7 @@
 }
 
 a[vr-item]:focus {
-    background-color: $primary;
```

Comment:
@aditi-tiwari17 @shalinee-walkover Please check this code revert

Reply:
checked, its fine